change/office-ui-fabric-react-2019-07-11-17-36-35-marygans-shimmerUpdates.json

```
@@ -0,0 +1,8 @@
+{
+  "comment": "Shimmer: Update palette references to semantic colors",
+  "type": "none",
```

I think this still needs to be patch.
```suggestion
  "type": "patch”,
```